src/components/Notifications/NoticeBoard.tsx

```diff
@@ -1,55 +1,129 @@
+import { useState } from "react";
+import React from "react";
 import { useTranslation } from "react-i18next";
 
 import CareIcon from "@/CAREUI/icons/CareIcon";
 
+import { Avatar } from "@/components/Common/Avatar";
 import Loading from "@/components/Common/Loading";
 import Page from "@/components/Common/Page";
 
 import routes from "@/Utils/request/api";
 import useTanStackQueryInstead from "@/Utils/request/useQuery";
-import { formatDateTime, formatName } from "@/Utils/utils";
+import { formatName, relativeTime } from "@/Utils/utils";
+
+import DialogModal from "../Common/Dialog";
+import { UserBareMinimum } from "../Users/models";
+import { NotificationData } from "./models";
 
 export const NoticeBoard = () => {
   const { t } = useTranslation();
   const { data, loading } = useTanStackQueryInstead(routes.getNotifications, {
     query: { offset: 0, event: "MESSAGE", medium_sent: "SYSTEM" },
   });
```

Comment:
switch to using `useQuery` instead

Reply:
@rithviknishad we are using useTanStackQueryInstead across the application. Why UseQuery.

Reply:
useTanStackQuery instead is just a proxy for existing usages since it's hard to migrate all of them at once. Tanstack Query was recently installed, until then we were using our own custom implementation.

Refer docs for more: https://docs.ohc.network/docs/care/development/data-fetching-in-care/